src/main/java/net/minecraftforge/network/ServerStatusPing.java

```diff
@@ -35,13 +37,19 @@
 import org.apache.logging.log4j.Logger;
 
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
+import java.util.Objects;
 import java.util.function.Supplier;
 
 import java.util.stream.Collectors;
 import java.util.stream.StreamSupport;
 
 /**
+ * Represents additional data sent by FML when a server is pinged.
+ * Previous versions used the following format:
+ * <code>
+ * <pre>
```

Comment:
This should use a `{@code}` block tag wrapped in `<pre>`:
```suggestion
 * <pre>{@code
```